phpBB/includes/ucp/ucp_pm_viewfolder.php

```diff
@@ -509,15 +535,38 @@ function get_pm_from($folder_id, $folder, $user_id)
 		$sql_sort_order = $sort_by_sql[$sort_key] . ' ' . $direction;
 	}
 
-	$sql = 'SELECT t.*, p.root_level, p.message_time, p.message_subject, p.icon_id, p.to_address, p.message_attachment, p.bcc_address, u.username, u.username_clean, u.user_colour, p.message_reported
-		FROM ' . PRIVMSGS_TO_TABLE . ' t, ' . PRIVMSGS_TABLE . ' p, ' . USERS_TABLE . " u
-		WHERE t.user_id = $user_id
+	$sql_ary = array(
+		'SELECT'	=> 't.*, p.root_level, p.message_time, p.message_subject, p.icon_id, p.to_address, p.message_attachment, p.bcc_address, u.username, u.username_clean, u.user_colour, p.message_reported',
+		'FROM'		=> array(
+			PRIVMSGS_TO_TABLE	=> 't',
+			PRIVMSGS_TABLE		=> 'p',
+			USERS_TABLE			=> 'u',
+		),
+		'WHERE'		=> "t.user_id = $user_id
 			AND p.author_id = u.user_id
 			AND $folder_sql
 			AND t.msg_id = p.msg_id
-			$sql_limit_time
-		ORDER BY $sql_sort_order";
-	$result = $db->sql_query_limit($sql, $sql_limit, $sql_start);
+			$sql_limit_time",
+		'ORDER'	=> $sql_sort_order,
+	);
+
+	/**
+	* Modify template variables before they are assigned
```

Comment:
What's the purpose of modifying sql after it is executed? The code above this comment just replaces string form of sql with array version.